tests/test.php

```diff
@@ -22,41 +22,42 @@
 
 echo "\n\nStarting tests\n";
 
-include_once("issuingBalance.php");
-include_once("issuingProduct.php");
-include_once("issuingCard.php");
-include_once("issuingCardLog.php");
-include_once("issuingHolder.php");
-include_once("issuingHolderLog.php");
-include_once("issuingInvoice.php");
-include_once("issuingInvoiceLog.php");
-include_once("issuingPurchase.php"); 
-include_once("issuingPurchaseLog.php");
-include_once("issuingTransaction.php");
-include_once("issuingWithdrawal.php");
-include_once("pixDomain.php");
-include_once("pixBalance.php");
-include_once("pixChargeback.php"); 
-include_once("pixChargebackLog.php");
-include_once("pixClaim.php"); 
-include_once("pixClaimLog.php");
-include_once("pixDirector.php");
-include_once("pixInfraction.php"); 
-include_once("pixInfractionLog.php");
-include_once("pixKey.php");
-include_once("pixKeyLog.php");
-include_once("pixRequest.php");
-include_once("pixRequestLog.php");
-include_once("pixReversal.php"); 
-include_once("pixReversalLog.php");
-include_once("pixStatement.php");
-include_once("dynamicBrcode.php");
-include_once("staticBrcode.php");
-include_once("creditNote.php");
-include_once("creditNoteLog.php");
-include_once("creditNotePreview.php");
-include_once("webhook.php");
-include_once("event.php");
-include_once("key.php");
+// include_once("issuingBalance.php");
+// include_once("issuingProduct.php");
+// include_once("issuingCard.php");
+// include_once("issuingCardLog.php");
+// include_once("issuingHolder.php");
+// include_once("issuingHolderLog.php");
+// include_once("issuingInvoice.php");
+// include_once("issuingInvoiceLog.php");
+// include_once("issuingPurchase.php"); 
+// include_once("issuingPurchaseLog.php");
+// include_once("issuingTransaction.php");
+// include_once("issuingWithdrawal.php");
+include_once("merchantCategory.php");
+// include_once("pixDomain.php");
+// include_once("pixBalance.php");
+// include_once("pixChargeback.php"); 
+// include_once("pixChargebackLog.php");
+// include_once("pixClaim.php"); 
+// include_once("pixClaimLog.php");
+// include_once("pixDirector.php");
+// include_once("pixInfraction.php"); 
+// include_once("pixInfractionLog.php");
+// include_once("pixKey.php");
+// include_once("pixKeyLog.php");
+// include_once("pixRequest.php");
+// include_once("pixRequestLog.php");
+// include_once("pixReversal.php"); 
+// include_once("pixReversalLog.php");
+// include_once("pixStatement.php");
+// include_once("dynamicBrcode.php");
+// include_once("staticBrcode.php");
+// include_once("creditNote.php");
+// include_once("creditNoteLog.php");
+// include_once("creditNotePreview.php");
+// include_once("webhook.php");
+// include_once("event.php");
+// include_once("key.php");
```

Comment:
Hmmm

Reply:
"How we write code in big tech companies"